firefox-ios/firefox-ios-tests/Tests/ClientTests/Frontend/Homepage/HomepageDiffableDataSourceTests.swift

```diff
@@ -8,13 +8,13 @@ import MozillaAppServices
 
 @testable import Client
 
+@MainActor
```

Comment:
curious why we add it here but not for other classes? i.e. `HomepageViewControllerTests`

Reply:
thanks for explaining 🙏  <3